src/Components/DisplayAllStudents.jsx

```diff
@@ -0,0 +1,37 @@
+import StudentCard from "../StudentCard";
+function DisplayAllStudents({
+  changeCohortDisplay,
+}) {
+  
+
+  return (
+    <main className="mainTotal">
+      <ul>
+        {changeCohortDisplay.map((data) => {
+          let month = new Date(data.dob).toLocaleString("en-US", {
+            month: "long",
+          });
+          let dobConstruct = data.dob.split("/");
+          let newBirth = `${month} ${dobConstruct[1]}, ${dobConstruct[2]}`;
+
+          let Track =
+            data.certifications.resume &&
+            data.certifications.linkedin &&
+            data.certifications.github &&
+            data.certifications.mockInterview &&
+            data.codewars.current.total > 600;
+
+          return (
+            <StudentCard
```

Comment:
You might wanna add a key for this, running into a bug where adding a note to the first student in a list carries over that note to other students in different cohorts